### ISSUE TITLE
Failed to format the disk refer this article for details.

### ISSUE THREAD

**Issue by Gavin123-code** [pocopico/tinycore-redpill#110]
Im running redpil VMDK variant on proxmox 7.2.3 and this error always occurs at 8%/Formatting system partition 
Network Card:VMware VMXnet3
Machine:q35
RAM:4096
Cores:2
SATA: All drives are sata
Proxmox device:A trash for this kinda thing dell inspiron 3470
DSM disk size:40G
Other drives: Not inserted yet but planning to have a extremely stupid 4 virtual 512GB disks

Redpill Stuff
Platform:Broadwellnk
DSM Version: 7.1.0-42661 
Updated: yes ran update and fullupgrade before starting
  
hopefuly this should be enough info to find the problem any support would be greatly appreciated
![Screenshot 2022-07-05 102739](https://user-images.githubusercontent.com/73368781/177351601-55917580-d39b-4c52-88dc-648413c78ff7.png)

**Comment by Gavin123-code** [comment]
fixed dont know how but i fixed it

**Comment by ner00** [comment]
I have the same issue except it doesn't even get past 0%, have found no way to fix it.

**Comment by ner00** [comment]
I have found the issue, and that's to to with the disk size, which I was leaving as 8GB for basic initial testing. After hours of searching I found this footnote on 

> _For DSM 7, new installs reserve 8GB for /dev/md0, 2GB for swap and the minimum volume size is 10GB.  This means the practical smallest usable VMDK is now 21GB._

source: https://xpenology.com/forum/topic/9394-installation-faq/?do=findComment&comment=89097

I expanded the disk size from 8GB to 25GB and the installation started, although after the first reboot during the setup the system just shut down on its own, so that still doesn't make it work for me but it's another issue probably.
